tests/test_routes_projects_agent_tasks.py

```
@@ -117,20 +120,163 @@ async def test_get_task(self, ctx):
+                f"/api/projects/{pid}/tasks/{tid}",
+                json={"parent_task_id": "some-parent"},
+                headers=_hdr(token),
+            )
+        assert resp.status_code == 403
+
+    async def test_non_lead_agent_patch_rejected(self, ctx):
+        """An agent that neither created nor leads the project cannot PATCH
+        its cards -> 403."""
+        pid = await _new_project(ctx, "alpha")
+        tid = await _new_task(ctx, pid)
+        cid, token = await _mint_agent(ctx, pid, scopes=("project_tasks_update",))
```

Ruff (0.16.0)

[warning] Unpacked variable `cid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)